src/components/FormAddBook.js

```diff
@@ -0,0 +1,63 @@
+import { React, useRef } from 'react';
+import { useDispatch } from 'react-redux';
+import { nanoid } from 'nanoid';
+import { addBookApi } from '../redux/books/books';
+import './FormAddBook.css';
+
+function FormAddBook() {
+  const titleInputRef = useRef();
+  const categoryInputRef = useRef();
+  const dispatch = useDispatch();
+
+  function submitBookToStore(event) {
+    event.preventDefault();
+    const newBook = {
+      item_id: nanoid(),
+      title: titleInputRef.current.value,
+      category: categoryInputRef.current.value,
+    };
+    dispatch(addBookApi(newBook));
+    titleInputRef.current.value = '';
+    categoryInputRef.current.value = '';
+  }
+
+  return (
+    <div className="add-book-container">
+      <h2 className="add-title">ADD NEW BOOK</h2>
+      <form onSubmit={submitBookToStore} className="flex justify-between">
+        <input
+          type="text"
+          name="name"
+          placeholder="Book Title"
+          id="title"
+          ref={titleInputRef}
+          required
+          className="input-title"
+        />
```

Comment:
- [ ] The book title is not updated after I click the submit button to add a book. It will be nice to fix this.

![image](https://user-images.githubusercontent.com/51296741/156841343-9ee10730-6c21-479d-8e2d-ed53fba30974.png)

Reply:
Hey bro, the Author is not part of the object the API, the API does not recieve the data.

Reply:
@zairdon20, my main concern is for the app to work as before. Currently, when I add a book, the book title is not updated. As you can see from the circled areas.

Reply:
OKay, I have made the change now thanks